etna/models/prophet.py

```diff
@@ -322,6 +322,9 @@ class ProphetModel(
 
     Notes
     -----
+    This model requires ``prophet`` extension to be installed.
```

Comment:
Can we highlight this block somehow? i.e like this 
<img width="715" alt="Снимок экрана 2023-07-31 в 09 23 27" src="https://github.com/tinkoff-ai/etna/assets/55380696/90a2822e-26d1-4547-9b06-b0d87e02c779">

Reply:
- `Notes` section isn't rendered anything special. I think it will be strange to give one note more attention than the others.
- We can add move notion about the extension into `warnings` section, but I'm not sure if it is ok by the meaning.
- We can move this notion to the exntended summary section and wrap it into `note` like on the picture that you've shown.

The info about meaning of the different sections is here: https://numpydoc.readthedocs.io/en/latest/format.html#extended-summary

Reply:
Ok, we can try to wrap this notion into `note` syntax: https://numpydoc.readthedocs.io/en/latest/format.html#other-points-to-keep-in-mind 
Let's try to do this.